services/batchDeleteUsers.js

```diff
@@ -0,0 +1,27 @@
+const {ddbClient, ddb} = require('../dbClient/ddbClient');
+const { CUSTOMER_ID, TABLE_NAME } = require('../shared/constants');
+
+const generateBatchDeleteUserParams = (emails) => {
+    return emails.map(email => {
+        return {
+            DeleteRequest: {
+                Key: {
+                    pk: `CUSTOMER_ID::${CUSTOMER_ID}`,
+                    sk: `EMAIL::${email}`,
+                }
+            }
+        };
+    });
+}
+
+const batchDeleteUsers = (emails) => {
+    const params = {
+        RequestItems: {
+            [TABLE_NAME]: generateBatchDeleteUserParams(emails)
+        }
+    };
+
+    return ddbClient.batchWrite(params).promise();
```

Comment:
Should handle error for the batchWrite : https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Programming.Errors.html#Programming.Errors.BatchOperations